src/mono/wasm/wasm.proj

```diff
@@ -71,7 +71,8 @@
       <_EmccCommonFlags Include="--source-map-base http://example.com" />
 
       <_EmccCommonFlags Include="-s STRICT_JS=1" />      
-      <_EmccCommonFlags Include="-s MODULARIZE=1" Condition="'$(WasmEnableES6)' != 'false'" />
+      <_EmccCommonFlags Include="-s EXPORT_NAME=&quot;'createDotnetRuntime'&quot;" />
+      <_EmccCommonFlags Include="-s MODULARIZE=1"/>
```

Comment:
It's intentional that this is no longer conditional, right? Because if ES6 isn't enabled we use a commonjs module instead?

Reply:
Exactly.